src/unstract/adapters/llm/ollama/src/static/json_schema.json

```diff
@@ -0,0 +1,46 @@
+{
+  "title": "Ollama AI LLM",
+  "type": "object",
+  "required": [
+    "base_url",
+    "model"
+  ],
+  "properties": {
+    "adapter_name": {
+      "type": "string",
+      "title": "Name",
+      "default": "",
+      "description": "Provide a unique name for this LLM adapter instance. Example: ollama-instance-1"
+    },
+    "model": {
+      "type": "string",
+      "title": "Model",
+      "default": "",
+      "description": "Provide the model name to be used. Example:llama2, llama3, mistral"
+    },
+    "base_url": {
+      "type": "string",
+      "title": "Base URL",
+      "default": "",
+      "description": "Provide the base URL where Ollama server is running."
+    },
+    "context_window": {
```

Comment:
Currently, we do send context_window value for every LLM by accessing the metadata. Pls refer to the adapter details API of LLMs. By taking the value here, should we make sure that the same value gets reflected there also? I am not very clear of the significance of setting the context window here, Also, is there any range within which the value needs to be? If yes, then I hope llama-index / the underlying ollama implementation throws a crisp error when we give a value outside the boundary.

Reply:
@gaya3-zipstack Ollama supports running of multiple models in local. This field must be needed to be sent based on the models chosen. Also, I am a bit confused here on the `adapter details API` mentioned. We can discussed this in details over a call.

Reply:
@gaya3-zipstack the user will know the context size as they are running the model themselves. And I think this is part of the constructor of the ollama client... @harini-venkataraman can you confirm?

Reply:
@arun-venkataswamy Yes, Llama index`s Ollama client has this in their model, and is sent to Ollama`s native client though kwargs. 
@gaya3-zipstack Since multiple models can be used and context_window will be different for each model, this field will be needed.

Reply:
I am fine with passing the context_window. Just mentioning that it will be good to make sure that the same window size gets returned in the adapter instance - GET API. @harini-venkataraman